swiftsimio/visualisation/slice.py

```diff
@@ -387,6 +80,10 @@ def slice_gas_pixel_grid(
         Particles outside of this range are still considered if their
         smoothing lengths overlap with the range.
 
+    backend : str, optional
+        Backend to use. Choices are "sph" (default) for interpolation using kernel weights or
+        "nearest_neighbours" for nearest neighbour interpolation.
+
     periodic : bool, optional
```

Comment:
Please make sure to have comment line lengths be consistent with code. I like to keep them under 80.